src/azul/files.py

```diff
@@ -8,9 +8,45 @@
 import os.path
 import tempfile
 from typing import (
+    Tuple,
     Union,
 )
 
+composite_extensions = [
+    'bam.bai',
```

Comment:
I thought `foo.bam` would be accompanied by a `foo.bai` not a `foo.bam.bai`. Please provide evidence that this composite extension is actually used.

Reply:
```
(azul.sc) daniel@Crispin ~/azul.sc $ grep 'bam.bai' files-* | head
files-prod-analysis.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/prod/analysis/data/00039bc5-86fc-4654-8ace-92eb6215155c/92da8d5e-e839-43a2-afce-e009b0b3a61f_qc.bam.bai
files-prod-analysis.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/prod/analysis/data/00103dd7-dd95-4ac0-9d24-29bb493a5179/e8862ff1-822f-460f-8aa0-5c4e7f2407ef_qc.bam.bai
files-prod-analysis.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/prod/analysis/data/0016d876-3768-4c76-8957-8804bb7208e8/0432e9a5-604f-4cb7-8571-014eb5fd8ba2_qc.bam.bai
files-prod-analysis.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/prod/analysis/data/001df33d-3ac3-4e85-b145-512c85f525f4/756b46c7-fc84-4175-ba03-7dc13c0819ed_qc.bam.bai
files-prod-analysis.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/prod/analysis/data/0021142e-e376-4f2c-9b0e-f96868c20834/6756419e-aa5c-46f6-9e00-a5983d8b6d6c_qc.bam.bai
files-prod-analysis.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/prod/analysis/data/0024233e-cdff-4d4c-ba36-82270fd8ef5c/0432e9a5-604f-4cb7-8571-014eb5fd8ba2_qc.bam.bai
files-prod-analysis.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/prod/analysis/data/002df41d-2327-4012-abbb-0401279256c8/a0800882-6b67-40a3-b10a-0d2d4ba7f9c1_qc.bam.bai
files-prod-analysis.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/prod/analysis/data/002e1bd5-e87e-4476-bab3-0bc3e3bf4584/74f154d0-736c-4c50-a472-29b9ce8c2d03_qc.bam.bai
files-prod-analysis.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/prod/analysis/data/0030f670-9838-45bd-9889-278ce5a2b955/d7c50d7f-0f1d-4ab1-9f96-aca48dbe2b22_qc.bam.bai
files-prod-analysis.txt:gs://broad-dsp-monster-hca-prod-ucsc-storage/prod/analysis/data/00368087-fc7e-4438-8381-2482ae2983ed/147692ca-e9c7-459c-a2c6-db72b48eed32_qc.bam.bai
(azul.sc) daniel@Crispin ~/azul.sc $ grep 'bam.bai' files-* | wc -l
   14178
```